frontend/src/action/auth-actions.js

```diff
@@ -1,37 +1,48 @@
 import superagent from 'superagent';
+import * as utils from '../lib/util';
 
-export const tokenSet = token => {
+/********************************************************************************
+*         Synchronous                                                           *
+********************************************************************************/
+
+export const login = token => {
   return {
-    type: 'TOKEN_SET',
+    type: 'LOGIN',
     payload: token,
   };
 };
 
-export const tokenDelete = () => {
+export const logout = () => {
+  utils.cookieDelete('chatToken');
   return {
-    type: 'TOKEN_DELETE',
+    type: 'LOGOUT',
   };
 };
 
+/********************************************************************************
+*        Asynchronous                                                           *
+********************************************************************************/
+
 export const signupRequest = user => dispatch => {
-  return superagent.post(`http://localhost:3000/register`)
+  return superagent.post(`http://localhost:3000/signup`)
```

Comment:
You should be using a global variable along the lines "API_URL" for these strings.